portal/v2/src/ClusterDetailList.tsx

```diff
@@ -70,6 +73,36 @@ export class ClusterDetailComponent extends Component<ClusterDetailComponentProp
             <MachineSetsWrapper currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded}/>
           );
         }
+        case "apistatistics":
+        {
+          return (
+            <Statistics currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded} statisticsType={"api"}/>
+          );
+        }
+        case "kcmstatistics":
+          {
+            return (
+              <Statistics currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded} statisticsType={"kcm"}/>
+            );
+          }
+        case "dnsstatistics":
+        {
+          return (
+            <>
+            <Statistics currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded} statisticsType={"dns"}/>
+            </>
+          );
+        }
+        case "ingressstatistics":
+        {
+          return (
+            <>
+            <Statistics currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded} statisticsType={"ingress"}/>     
+            </>
```

Comment:
No need for fragments `<></>` around these two Components.

Reply:
done